README.md

```diff
@@ -11,6 +15,21 @@ To start this app:
 
 After that, you can visit [`localhost:4000`](http://localhost:4000) from your browser, and play with it.
 
+### With Docker
+
+First you need to build the *phoenix* image by running
+
+`docker build -t phoenix docker`
+
+Then you can run the application
+
+```bash
+cd docker
+docker-compose up -d
+```
+
+It will take awhile for the first deploy as it install all dependencies for the projet. Visit [`localhost:4000`](http://localhost:4000) from your browser, the page will automatically be reloaded when the application starts.
```

Comment:
Pour moi, tu pourrai juste dire "For the first deployment, it can take a while."
Aussi, un `double space` entre la 1ère et 2ème phrase ;-)